airflow_spark_on_k8s_job_builder/core.py

```diff
@@ -439,6 +442,49 @@ def set_secrets(self, conf: Dict[str, Union[str, int, float]]) -> "SparkK8sJobBu
         self.get_job_params()["executor"]["secrets"].update(conf)
         return self
 
+    def add_global_persistent_volume(
+            self,
+            volume_name: str,
+            claim_name: str,
+            mount_path: str,
+            readonly: bool = False,
+    ) -> "SparkK8sJobBuilder":
```

Comment:
yet another minor nit: I know you didn't declare any of these params as type `Optional[str]`, but that doesn't mean someone can't inject a null, and that the compiler will catch it. So suggestion to add extra explicit emergency lever to make sure no funkyness is introduced at runtime:
```
if any(param is None for param in [volume_name, claim_name, mount_path]):
        raise ValueError("None of the string parameters can be None")
```